src/roles/webfilter-deploy/tasks/main.yml

```diff
@@ -111,16 +118,20 @@
       timeout_seconds: 600
       test_interval_seconds: 10
 
-  - name: Run incompass operation command for seed
-    shell: /opt/vsd/webfilter/scripts/incompassOperations.sh -a all -p /opt/vsd/webfilter/seed_34
+  - block:
 
-  - name: Run incompass operation command
-    shell: /opt/vsd/webfilter/scripts/incompassOperations.sh -a all
+    - name: Run incompass operation command for seed
+      shell: /opt/vsd/webfilter/scripts/incompassOperations.sh -a all -p /opt/vsd/webfilter/seed_34
 
-  - name: Wait for webfilter processes to become running
-    monit_waitfor_service:
-      name: "{{ webfilter_proc['state'].keys() }}"
-      timeout_seconds: 600
-      test_interval_seconds: 10
+    - name: Run incompass operation command
+      shell: /opt/vsd/webfilter/scripts/incompassOperations.sh -a all
+  
+    - name: Wait for webfilter processes to become running
+      monit_waitfor_service:
+        name: "{{ webfilter_proc['state'].keys() }}"
+        timeout_seconds: 600
+        test_interval_seconds: 10
+
+    when: run_incompass_operation is defined
```

Comment:
this should be "run_incompass_operation is defined and run_incompass_operation". As it can be defined and False as well